src/test/kotlin/org/rust/RsTestBase.kt

```diff
@@ -220,8 +222,11 @@ abstract class RsTestBase : LightPlatformCodeInsightFixtureTestCase(), RsTestCas
         }
     }
 
-    protected inline fun <reified T : PsiElement> findElementInEditor(marker: String = "^"): T {
-        val (element, data) = findElementWithDataAndOffsetInEditor<T>(marker)
+    protected inline fun <reified T : PsiElement> findElementInEditor(marker: String = "^"): T =
+        findElementInEditor(T::class.java, marker)
+
+    protected fun <T : PsiElement> findElementInEditor(psiClass: Class<T>, marker: String): T {
```

Comment:
It makes debugging & stack trace analyzing more complicated. As I remember, I made these changes exactly when I too tired to debug the `findElementsWithDataAndOffsetInEditor` function (you can see some changes there) when it is inlined. I think that we _really_ need these changes, especially in tests, because it's a something that usually fails and we are often forced to analyze stack traces from there or debug it, so it should be as simple as possible.